src/sentry/tasks/base.py

```diff
@@ -125,7 +126,11 @@ def _wrapped(*args, **kwargs):
                 metrics.timer(key, instance=instance),
                 track_memory_usage("jobs.memory_change", instance=instance),
             ):
-                result = func(*args, **kwargs)
+                try:
+                    result = func(*args, **kwargs)
+                except SoftTimeLimitExceeded:
+                    capture_exception(level="warning")
```

Comment:
I think this error will already be caught when we raise, so isn't this double sending now? Would it be more helpful to have a stat instead?

Reply:
ah good point. i didn't think to look in Sentry. we indeed already have [them](https://sentry.sentry.io/issues/?project=1&query=SoftTimeLimit&referrer=issue-list&sort=date&statsPeriod=24h&viewId=50632).  i'd rather not introduce another high cardinality metric if possible. i can set up some sentry alerts for this if that works.